### PR TITLE
Clippyfy rust 1.78.0

### PR DESCRIPTION
**Pull request by pythcoiner**
Note [this](https://github.com/pythcoiner/liana/blob/7a11fe584a3422502dce8519700dfb05c6f7f937/gui/src/main.rs#L5-L6) is related to a clippy warning that appear on MacOs (and i guess on windows)

### CODE REVIEW

**Comment by jp1ac4** [comment]
This now needs a rebase.

**Comment by pythcoiner** [comment]
> This now needs a rebase.
done

**Comment by pythcoiner** [comment]
> This now needs a rebase.

done

File: gui/src/app/state/psbt.rs [hunk 1]
```diff
@@ -35,6 +35,7 @@ use crate::{
 };
 
 pub trait Action {
+    #[allow(dead_code)]
```
Comment (darosior) [inline gui/src/app/state/psbt.rs:38]: Why would we keep dead code around?

Comment (edouardparis) [reply gui/src/app/state/psbt.rs:38]: yes

Comment (pythcoiner) [reply gui/src/app/state/psbt.rs]: done

**Review by edouardparis — APPROVED** [review]
utACK a8e256eb6b5463dfd32599618998592a94835193

**Pull request merged**